packages/terra-table/docs/README.md

```diff
@@ -0,0 +1,72 @@
+# Terra Table
+
+The Terra Table is a structural component to arrange content within table.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-table`
+  - `yarn add terra-table`
+
+## Usage
+
+### Table
+```jsx
+import React from 'react';
+import Table from 'terra-table';
+
+<Table isStriped={false}>
+  <Table.Header>
+    <Table.HeaderContent display={'Column Heading 1'} key={1} minWidth={'small'} />
```

Comment:
The examples are using a `content` prop instead of `display`. Should this be updated to switch `display` prop to `content`